macros/REST_MakeProcess.C

```diff
@@ -128,17 +128,17 @@ Int_t REST_MakeProcess(TString name, TString inputevent = "TRestEvent", TString
     headerfile << "    Double_t fDummyVar = 3.14; //!" << endl;
     headerfile << endl;
     headerfile << "public:" << endl;
-    headerfile << "    any GetInputEvent() { return " << inputeventname << "; }" << endl;
-    headerfile << "    any GetOutputEvent() { return " << outputeventname << "; }" << endl;
+    headerfile << "    any GetInputEvent() const override { return " << inputeventname << "; }" << endl;
+    headerfile << "    any GetOutputEvent() const override { return " << outputeventname << "; }" << endl;
     headerfile << endl;
-    headerfile << "    void InitProcess();" << endl;
+    headerfile << "    void InitProcess() override;" << endl;
     headerfile << endl;
```

Comment:
```suggestion
    headerfile << endl;
    headerfile << "const char* GetProcessName() override const { return \"CHANGEME!\"; }" << endl;
    headerfile << endl;
```

Reply:
I think before `GetProcessName` was not pure virtual.

Reply:
That's weird, I have compiled a dummy library in the code without this line and without any isse.

Btw, `override` should be placed after the `const` --> `const char* GetProcessName() const override`

Reply:
Thats surprising, sure it was a process?

Reply:
Apparently it gives runtime error but not compilation error...